Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -12,6 +12,8 @@
 import cutsRecoTracks_cfi
 
 from SimTracker.TrackerHitAssociation.tpClusterProducer_cfi import *
+from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
+trackingPhase2PU140.toModify(tpClusterProducer, pixelSimLinkSrc = "simSiPixelDigis:Pixel")
```

Comment:
Proper place for customizing `tpClusterProducer` would be in
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/TrackerHitAssociation/python/tpClusterProducer_cfi.py
but actually it is not needed since @lgray already added the customization (for `phase2_tracker`).